packages/governance/src/committeeRegistrar.js

```diff
@@ -19,6 +37,7 @@ const start = zcf => {
       const { publicFacet } = allQuestions.get(key);
       return [E(publicFacet).isOpen(), key];
     });
+
     const isOpenQuestions = await allComparable(harden(isOpenPQuestions));
```

Comment:
I have no idea if it is even possible to type `allComparable` so that TS can derive an output type from the input type. So providing it manually.
```suggestion
    /** @type {[boolean, Handle<'Ballot'>][]} */
    const isOpenQuestions = await allComparable(harden(isOpenPQuestions));
```

Reply:
done